mongodb/mongodb.rb

```diff
@@ -404,46 +555,46 @@ def monitor_service(service, metric_group)
   raise e
 end
 
-
 @services.each do |service|
```

Comment:
The installer script has been updated to add both services, instead of agent manipulating the config on its own